www/src/app/shared/auth/auth.service.ts

```diff
@@ -1,19 +1,23 @@
 import {Injectable} from "@angular/core";
 import {Router} from "@angular/router";
-import {Principal} from "./principal.service";
+
+import {Principal} from "./principal-storage.service";
 import {StateStorageService} from "./state-storage.service";
+import {PrincipalService} from "./principal.service";
 
+// TODO: MD - Deprecated - remove this file
```

Comment:
Might as well just remove it then. It's just imported and provided in www/src/app/shared/shared.module.ts